### DISCUSSION TITLE
Provide capability to no longer be somebody's successor

### DISCUSSION THREAD

**Discussion by tomkerkhove** [community/community#4945] [Other Feature Feedback, Questions, & Ideas]
It would be nice if you could let somebody know you no longer want/can be their successor.

This is equally important as setting it up because you commit to taking care of somebody’s hard work when he/she passes. But what if you are no longer in touch and don’t feel comfortable with it anymore? Then it’s best to let that person know, instead of ignoring your commitment at all.

Another example is if your successor is your partner, but you got a divorce.

**Comment by Salman-dott** [comment]
I think you just need to talk to a person and separate these responsibilitites, exactly as you would be separating everything else. Even when you just chat [there](https://www.together2night.com/local-hookup.html) with someone without any obligations, you need to set up boundaries. And when it comes to closer relations, it is much more important. Just say it simple. Because we are no longer close, I would like to withdraw my promise to take care of this and that.
